vision_agent/tools/tools.py

```diff
@@ -420,21 +481,70 @@ def florencev2_roberta_vqa(prompt: str, image: np.ndarray) -> str:
 
     Example
     -------
-        >>> florencev2_roberta_vqa('What is the top left animal in this image ?', image)
+        >>> florence2_roberta_vqa('What is the top left animal in this image?', image)
         'white tiger'
     """
 
     image_b64 = convert_to_b64(image)
     data = {
         "image": image_b64,
         "question": prompt,
-        "function_name": "florencev2_roberta_vqa",
+        "function_name": "florence2_roberta_vqa",
     }
 
     answer = send_inference_request(data, "florence2-qa", v2=True)
     return answer  # type: ignore
 
 
+def ixc25_image_vqa(prompt: str, image: np.ndarray) -> str:
+    """'ixc25_image_vqa' is a tool that can answer any questions about arbitrary images
+    including regular images or images of documents or presentations. It returns text
+    as an answer to the question.
```

Comment:
NIT: Ww might need to change the description based on how often agent picks this tool. I don't know what the right description as of now, so lets start with this. However we might have to do add something like "The tool cannot perform detection or segmentation tasks.... " This way other tools are invoked properly.

Reply:
Yeah need to benchmark this, so in this PR I've replaced florence2 vqa with this tool